src/utils/search.ts

```diff
@@ -1,15 +1,22 @@
 import { toLower } from 'lodash';
 import { matchSorter } from 'match-sorter';
 
-export const filterList = (list, searchQuery, keys = null, options = null) =>
+export const filterList = (
+  list: any[],
+  searchQuery: string,
+  keys = null,
+  options = null
+) =>
   matchSorter(list, searchQuery, {
     keys,
+    // @ts-expect-error ts-migrate(2698) FIXME: Spread types may only be created from object types... Remove this comment to see the full error message
     ...options,
   });
 
-export const filterScams = (safeList, nonSafeList) => {
+export const filterScams = (safeList: any, nonSafeList: any) => {
```

Comment:
I could be wrong but I don't actually see where this function is used. It is exported from `src/utils` but I don't see it imported anywhere. I might be missing something, though.